package.json

```diff
@@ -807,16 +807,17 @@
   "scripts": {
     "prepare": "husky install",
     "vscode:prepublish": "npm run panel:lineage && webpack --mode production",
-    "webpack": "npm run panel:lineage && webpack --mode development",
+    "webpack": "rm -Rf dist; npm run panel:lineage && webpack --mode development",
     "webpack-dev": "npm run panel:lineage && webpack --mode development --watch",
-    "test-compile": "tsc -p ./",
     "lint": "eslint src --ext ts",
     "lint:fix": "eslint src --ext ts --fix",
     "deploy-vscode": "vsce publish",
     "build-vsix": "vsce package",
     "deploy-openvsx": "ovsx publish",
     "panel:lineage": "npm run build --prefix ./new_lineage_panel",
-    "install:panels": "npm install --prefix ./new_lineage_panel"
+    "install:panels": "npm install --prefix ./new_lineage_panel",
+    "pretest": "rm -Rf ./.vscode-test/extensions;rm -Rf dist; rm -Rf ./new_lineage_panel/dist; npm run panel:lineage; tsc -p ./",
```

Comment:
maybe we could have additional clean commands like, `clean:extension`, `clean:panel`, etc. and use those.